charter.md

```diff
@@ -0,0 +1,210 @@
+# observe-k8s
+
+## Mission Statement
+
+For cloud native workloads, observability is a key requirement, yet it’s a broad term and there are many tools and approaches available to achieve it.
+
+While the [whitepaper](https://github.com/cncf/tag-observability/blob/main/whitepaper.md) by the [CNCF TAG Observability](https://github.com/cncf/tag-observability) covers the theoretical foundations of modern observability, practitioners need guidance on how to achieve it in real life scenarios.
+
+This working group aims to provide:
+
+- a suite of realistic sample applications, which employ observability tools & techniques to demonstrate how practitioners can gain insight and understanding.
+- actionable, good practice & peer reviewed guidance on how to enable observability for real-life workloads. Observed workloads will be CNCF Projects.
```

Comment:
> Observed workloads will be CNCF Projects
what is ment by this exactly?

Reply:
That we try to focus for the system under observation (SuO) on CNCF projects. For example, a Kubernetes workload would be in scope, some proprietary provider API for running the workloads less so.

Reply:
Understood :) Thanks @mhausenblas - id be interested in pushing the envelope her from an SRE persona where running a single kube cluster VS a fleet changes observavility requirements (if this is something observe-k8s would be interested in)